Sources/LLVMWrappers/include/LLVMWrappers.h

```diff
@@ -0,0 +1,16 @@
+#ifndef LLVMWrappers_h
+#define LLVMWrappers_h
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void *_Nullable LLVMCreateOrcMCJITReplacement(void *_Nonnull module,
```

Comment:
This is pretty much copy-pasted from Trill. This is just so we can avoid including the LLVM-C headers as well.